lib/main.dart

```diff
@@ -36,6 +36,250 @@ class MyHomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(body: Center(child: Text("Portfolio Coming Soon!", style: Theme.of(context).textTheme.headline1!.copyWith(fontFamily: 'PalmerLake', color: const Color.fromRGBO(56, 88, 78, 1)),)));
+    return Scaffold(
+        body: Center(
+      child: Expanded(
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              SizedBox(
+                width: 1050,
+                height: 40,
+                child: Row(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    mainAxisAlignment: MainAxisAlignment.start,
+                    children: [
+                      SizedBox(
+                        width: 150,
+                        height: 10,
+                        child: Text(
+                          "  Equity Gals",
+                          style: Theme.of(context)
+                              .textTheme
+                              .headline4!
+                              .copyWith(
+                                  fontFamily: 'PalmerLake',
+                                  color: const Color.fromRGBO(56, 88, 78, 1)),
+                        ),
+                      ),
+                      SizedBox(width: 1300, height: 10, child: Container()),
+                      SizedBox(
+                        width: 100,
+                        height: 70,
+                        child: Text(
+                          "MENU",
+                          style: Theme.of(context)
+                              .textTheme
+                              .headline5!
+                              .copyWith(
+                                  fontFamily: 'Impact',
+                                  color: const Color.fromRGBO(242, 11, 249, 1)),
```

Comment:
This should be a variable as its used multiple times

Reply:
G'day noob, I have fixed all of these doubled variables and have created our own.